projects/plugins/jetpack/extensions/blocks/subscriptions/view.scss

```diff
@@ -96,3 +96,32 @@
 		}
 	}
 }
+
+.wp-block-jetpack-subscriptions__newsletter-categories {
+	margin-top: 8px;
+	margin-bottom: 24px;
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap;
+	gap: 8px;
+
+	.wp-block-jetpack-subscriptions__newsletter-category {
+		padding: 6px 10px;
+		font-family: "SF Pro Text", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+		font-size: 14px;
+		font-weight: 500;
+		line-height: 20px;
+		color: var(--studio-gray-50);;
+		border: 1px solid var(--studio-gray-5);
+		border-radius: var(--border-radius);
```

Comment:
I wonder if we could use WordPress colors here, for more consistency?

Here are some examples:
https://github.com/Automattic/jetpack/blob/3a3c4adc1c0913bac224ddb832713bd6101fe1ab/projects/plugins/jetpack/extensions/blocks/donations/view.scss#L28